L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -112,8 +112,13 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         waitForExistence(app.cells["Logins"], timeout: 15)
         app.cells["Logins"].tap()
 
-        waitForExistence(app.navigationBars.buttons.firstMatch, timeout: 15)
-        app.otherElements.buttons.element(boundBy: 2).tap()
+        // First time only: The message "Your passwords are now protected
+        // by Face ID..." is present when the Firefox app is run after the
+        // simulator has been erased.
+        waitForNoExistence(app.navigationBars["settings"])
+        if app.otherElements.buttons["Continue"].exists {
```

Comment:
No, it does not because that Continue button does not have an id. These tests run on different languages, so we need to be sure that they work using ids or a different way to access the elements but not labels as those change with the locale

Reply:
This is what works for me:
```
waitForExistence(app.navigationBars.element(boundBy: 0), timeout: 3)
waitForExistence(app.otherElements.buttons.element(boundBy: 2))
app.otherElements.buttons.element(boundBy: 2).tap()
```
The passwords navigation bar and the Continue button change with the locale, so you need to call them using other methods...